transactor/src/main/scala/io/mediachain/transactor/Dummies.scala

```diff
@@ -12,6 +13,10 @@ object Dummies {
     }
     override def hashCode = num
     override def toString = "dummy@" + num
+
+    val CBORType = "dummyReference"
```

Comment:
I think it's ok to only support naked for now, things still seem not 100% shaken out in IPLD land